templates/windows/config/daemon.json.erb

```diff
@@ -0,0 +1,31 @@
+{
+    <%# TODO: get "authorization-plugins": [],%>
```

Comment:
Can we remove any commented code, just so there's nothing WIP sitting in master?

Reply:
sure